test/EVMHost.cpp

```diff
@@ -97,34 +97,28 @@ EVMHost::EVMHost(langutil::EVMVersion _evmVersion, evmc::VM& _vm):
 	{
 		evmc::address address{};
 		address.bytes[19] = precompiledAddress;
-		// 1wei
-		m_state.accounts[address].balance.bytes[31] = 1;
+		auto account = accounts[address];
+		if (is_zero(account.balance))
```

Comment:
Are the balances set before?

Reply:
Ah right, this is the constructor, forgot that.